OfflineService/OfflineDataRegionTableViewCell.swift

```diff
@@ -1,5 +1,5 @@
 import UIKit
-import MapboxCommon
+import MapboxCoreNavigation
 
 class OfflineDataRegionTableViewCell: UITableViewCell {
```

Comment:
In current implementation both last updated and pack size values are not visible when they're not downloaded, can we show them to match Android functionality. I think it's also important to show last updated value because it's used as tilesVersion for offline navigation.

Reply:
fixes